graphene_django/utils/utils.py

```diff
@@ -58,17 +92,14 @@ def maybe_queryset(value):
 
 
 def get_model_fields(model):
-    local_fields = [
-        (field.name, field)
-        for field in sorted(
-            list(model._meta.fields) + list(model._meta.local_many_to_many)
-        )
-    ]
-
-    # Make sure we don't duplicate local fields with "reverse" version
-    local_field_names = [field[0] for field in local_fields]
+    """
+    Gets all the fields and relationships on the Django model and its ancestry.
+    Prioritizes local fields and relationships over the reverse relationships of the same name
+    Returns a tuple of (field.name, field)
+    """
+    local_fields = get_local_fields(model)
+    local_field_names = {field[0] for field in local_fields}
```

Comment:
I changed this to a set, but I can totally change it back to list to not break anything. It looks like `get_reverse_fields` is public so I didn't want to mess with the signature too much.

Reply:
Seems like that was the right move to me, since you were able to keep the `get_reverse_fields` signature the same but improve lookup time-complexity for this internal use-case in `get_model_fields`, nice!